Sources/StreamChat/Workers/ReadStateSender.swift

```diff
@@ -4,74 +4,96 @@
 
 import Foundation
 
-@available(iOS 13.0, *)
-extension Chat {
-    actor ReadStateSender {
-        private let cid: ChannelId
-        private let authenticationRepository: AuthenticationRepository
-        private let messageRepository: MessageRepository
-        private let channelUpdater: ChannelUpdater
-        
-        init(
-            cid: ChannelId,
-            channelUpdater: ChannelUpdater,
-            authenticationRepository: AuthenticationRepository,
-            messageRepository: MessageRepository
-        ) {
-            self.authenticationRepository = authenticationRepository
-            self.cid = cid
-            self.channelUpdater = channelUpdater
-            self.messageRepository = messageRepository
+final class ReadStateSender {
```

Comment:
Moved controller's logic to this type which I created for the state layer. Now both use the same logic.